domains/auth/Auth.API/API/Program.cs

```diff
@@ -174,7 +173,7 @@
 }
 catch (Exception e)
 {
-    app.Logger.LogError(e, "An exception has occurred while starting up.");
+    app.Logger.LogError(e, "An exception has occurred while starting up");
```

Comment:
Why this change?

Reply:
To remove a warning. Compiler Hates ".", in error messages